docs/7_1/components/spinner.md

```diff
@@ -35,6 +35,8 @@ min | null | Double | Minimum boundary value
 max | null | Double | Maximum boundary value
 prefix | null | String | Prefix of the input
 suffix | null | String | Suffix of the input
+decimalSeparator | Default is taken from the view's locale | String | Decimal separator char.
+thousandSeparator | Default is taken from the view's locale | String | Thousand separator char.
```

Comment:
I would prefer `groupingSeparator` instead of `thousandSeparator`, see `DecimalFormatSymbols`.

Reply:
we use "thousandSeparator" in p:inputNumber. Using "thousandSeparator" there and "groupingSeparator" here is confusing.

Reply:
Agreed, should be consistent. Didn't know that.